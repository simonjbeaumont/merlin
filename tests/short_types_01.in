["reset","name","test.ml"]
["tell","struct","type t = int -> unit\nlet x = 5      \n"]
["tell","end","when lol\n"]
["tell","end",null]
["seek","exact",{"line":2,"col":4}]
["type","enclosing",{"line":2,"col":4}]
["seek","before",{"line":4,"col":0}]
["tell","struct","\nlet x = 5      \n"]
["tell","end","when lol\n"]
["tell","end",null]
["seek","exact",{"line":2,"col":4}]
["type","expression","x","at",{"line":2,"col":4}]
["seek","before",{"line":4,"col":0}]
["tell","struct","\nlet x = 5      \n"]
["tell","end","when lol\n"]
["tell","end",null]
["seek","exact",{"line":2,"col":4}]
["type","at",{"line":2,"col":4}]
