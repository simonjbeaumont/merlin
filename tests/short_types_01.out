["return",{"line":1,"col":0}]
["return",false]
["return",false]
["failure","invalid arguments"]
["return",[0,[]]]
["return",{"line":1,"col":0}]
["return",false]
["return",false]
["failure","invalid arguments"]
["error",{"type":"type","start":{"line":1,"col":0},"end":{"line":1,"col":1},"valid":true,"message":"Unbound value x"}]
["return",{"line":1,"col":0}]
["return",false]
["return",false]
["failure","invalid arguments"]
["exception","Not_found"]
